src/components/Editor/HighlightLine.js

```diff
@@ -91,6 +126,7 @@ export class HighlightLine extends PureComponent<Props> {
 }
 
 export default connect(state => ({
+  isResumingOrStepping: getIsResumingOrStepping(state),
```

Comment:
there should be a better term for this... `getIsResumingOrStepping` maybe `isRunning` dunno... @jimblandy @darkwing what do you think?